### ISSUE TITLE
Dutch: Missing comma after penultimate editor for @inbook

### ISSUE THREAD

**Issue by LukeSerne** [plk/biblatex-apa#221]
**Description**
In the Dutch localisation, there is no comma printed after the penultimate editor for `@inbook` (and related types). This is required according to [this example](https://libguides.uvt.nl/apa-handleiding/boeken#s-lib-ctab-15835140-0) in the Dutch localisation of APA7 by Tilburg University, which has been recreated in simplified form (to reduce the noise) in the example below.

**MWE**
```latex
\documentclass{report}

\usepackage[dutch]{babel}
\usepackage[style=apa, backend=biber]{biblatex}

\begin{filecontents}[overwrite]{test.bib}
@inbook{Simonton2019,
    author = {Simonton, Dean Keith},
    date = {2019},
    title = {Title},
    editor = {Subotnik, Rena F. and Olszewski-Kubilius, Paula and Worrell, Frank C.},
    booktitle = {Booktitle},
    pages = {201--224},
    publisher = {Publisher},
    doi = {10.1037/0000120-010},
}
\end{filecontents}
\addbibresource{test.bib}

\begin{document}
\nocite{*}
\printbibliography
\end{document}
```
**Observed output**
Simonton, D. K. (2019). Title. In R. F. Subotnik, P. Olszewski-Kubilius & F. C. Worrell (Red.), _Booktitle_ (pp. 201–224). Publisher. https://doi.org/10.1037/0000120-010

**Expected output**
Simonton, D. K. (2019). Title. In R. F. Subotnik, P. Olszewski-Kubilius, & F. C. Worrell (Red.), _Booktitle_ (pp. 201–224). Publisher. https://doi.org/10.1037/0000120-010

Note the comma after `P. Olszewski-Kubilius`.

When comparing this with the example on the webpage linked above, there is one other nonconformity (namely `Red.` instead of `Reds.`). However, this is easily fixable by redefining the `editors` bibliography string, so it's outside the scope of this issue. Several other bibliography strings need to be redefined to fully match the examples on the website of Tilburg University, which I plan on submitting in a separate issue.

**Comment by LukeSerne** [comment]
I can confirm this is fixed. Thanks!

**Comment by moewew** [comment]
I have similar feelings about this as I have about https://github.com/plk/biblatex-apa/issues/220. The example suggests Tilburg University want an Oxford comma here, but I'd have thought the Oxford comma is unusual in Dutch and I'm not sure Tilburg Univ. understand the implications here and really want all series to have a comma before the last item even with "and".

**Comment by LukeSerne** [comment]
I'd like to refer to my comment on #220 for my general stance on the situation with various competing standards for Dutch APA7. While I mentioned there are typos in the manual in the comment on the other issue, I do not think this is a typo - it is repeated far to often for that. Additionally, while the Oxford comma is unusual in Dutch, it is used in author lists as well, and while not explicitly mentioned that it applies to the editor list as well, the explanation for the editor list is [grouped with the "author" list](https://libguides.uvt.nl/apa-handleiding/elementen-van-referenties#s-lib-ctab-15826341-0), so they probably consider them to be related enough to have both use an Oxford comma.

The SURF manual is inconsistent about it. On page 65, they do not use the Oxford comma after an editors list (while there is [an erratum](https://www.auteursrechten.nl/erratum) for that example, it does not introduce an Oxford comma) in the middle example, but on the next 2 examples just a few lines below and on the next page, they *do* use an Oxford comma. There are also an example on page 78 using the Oxford comma, and no further examples not using the Oxford comma in an editor list following an `In `.

Also note that the "and" is an ampersand sign, and not the text `and` (or `en`) for both lists, further supporting the idea that the editor list is not seen as "normal text", but instead more like the list of authors at the beginning of an entry.

**Comment by moewew** [comment]
Not that it really helps to clear things up, but there is at least one interpretation (to which the current implementation of `biblatex-apa` subscribes) in which the comma before "&" in the author list (with reversed name format "Family, G.") is not actually an Oxford comma. See https://github.com/plk/biblatex-apa/issues/191, https://github.com/plk/biblatex-apa/issues/48.

**Comment by LukeSerne** [comment]
I got in contact with one of the authors of the SURF manual. He explained to me that the SURF manual follows the original APA7 guidelines on this, which (apparently) are that there should not be a comma if there are only 2 editors, but there should be a comma if there are 3 or more editors.

So, the MWE below should give these two lines as output (which it does on latest `master`)
```
Author, A. (2022). Two Editor Test. In O. Editor & T. Editor (Red.).
Author, A. (2023). Three Editor Test. In O. Editor, T. Editor, & T. Editor (Red.).
```

```latex
\documentclass{report}

\usepackage[dutch]{babel}
\usepackage[style=apa, backend=biber]{biblatex}

\begin{filecontents}[overwrite]{test.bib}
@inbook{TwoEditorTest,
    author = {Alex Author},
    editor = {Editor, One and Editor, Two},
    date = {2022},
    title = {Two Editor Test},
}

@inbook{ThreeEditorTest,
    author = {Alex Author},
    editor = {Editor, One and Editor, Two and Editor, Three},
    date = {2023},
    title = {Three Editor Test},
}
\end{filecontents}
\addbibresource{test.bib}

\begin{document}
\nocite{*}
\printbibliography
\end{document}
```

**Comment by plk** [comment]
This is already implemented in the APA style, yes (section 9.8).
